lib/read_table_A.pl

```diff
@@ -1103,19 +1081,6 @@ sub read_table_A {
             $code .= "use Philips_Hue;\n";
         }
     }
-    elsif ($type eq "PHILIPS_LUX"){
```

Comment:
How do I accomplish this task of pulling down the latest?

To create my branches, I am sure I started from scratch with a new fork and new branches; so I don’t understand how I don’t have the latest.

From: Lieven Hollevoet [mailto:notifications@github.com]
Sent: Sunday, October 25, 2015 4:45 PM
To: hollie/misterhouse <misterhouse@noreply.github.com>
Cc: Timothy Spaulding <spaulding@icanbrew.com>
Subject: Re: [misterhouse] New Dsc module (#550)


In lib/read_table_A.pl<https://github.com/hollie/misterhouse/pull/550#discussion_r42950129>:

> @@ -1103,19 +1081,6 @@ sub read_table_A {

>              $code .= "use Philips_Hue;\n";

>          }

>      }

> -    elsif ($type eq "PHILIPS_LUX"){

Hey Timothy,

could you please update the pull request so that the PHILIPS_LUX support is not removed?

Thank you,
Lieven.

—
Reply to this email directly or view it on GitHub<https://github.com/hollie/misterhouse/pull/550/files#r42950129>.